book/networking/p2p/network/README.md

```diff
@@ -0,0 +1,69 @@
+# Network
+
+TODO: Some high-level overview of peer-to-peer networking, either here or in a short chapter one level up in the tree for `p2p`
+
+The `network` crate is responsible for managing the node's connection to the Ethereum peer-to-peer (P2P) network, enabling communication with other nodes via the [various P2P subprotocols](https://github.com/ethereum/devp2p).
+
+TODO: Link to the "bird's eye docs" in `crates/net/network/src/lib.rs`?
```

Comment:
Maybe we should convert the mermaid docs to pngs so we can link to them easier?

Reply:
Yup, can do - I'm getting ahead of myself but it would be cool if we could also `ANCHOR` them so if their contents change, it would update here every time the book is re-built... but I haven't found a clean way to do that, so exporting as PNG is a good move right now

Reply:
sounds good, i can just do this through mermaid.live and embed into the repo